src/internal/connector/exchange/service_iterators.go

```diff
@@ -307,10 +311,14 @@ func FetchContactIDsFromDirectory(
 		Contacts().
 		Delta()
 
+	if len(oldDelta) > 0 {
```

Comment:
this needs to remain commented out until we have all the other pieces for delta token-based incrementals in place. Otherwise we'll end up with backups that only contain the changes

Reply:
Heh, yeah, I realized that over the weekend, too.